src/site/content/en/blog/speedy-css-tip-animated-loader/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Speedy CSS Tip! Animated Loader
+subhead: Let's make an animated CSS loader with scoped custom properties and animation-timing-function
+date: 2022-09-22
+hero: image/Dyx9FwYgMyNqy1kMGx8Orz6q0qC3/vzvc5Ep1blYUQMYYnAKT.jpg
+authors:
+  - jheyy
+tags: 
+  - blog
+  - css
+---
+Hop over to [CodePen](https://pen.new) and create a new pen.
```

Comment:
```suggestion
---

Hop over to [CodePen](https://pen.new) and create a new pen.
```

Reply:
Total nit, but a line between the front matter and the post is just a bit cleaner to read.

Reply:
All good. I removed it thinking it had added extra whitespace but don't think that was the case 😅